build.gradle.kts

```diff
@@ -0,0 +1,112 @@
+plugins {
+    idea
+    `java-library`
+    `maven-publish`
+    id("net.kyori.indra") version "3.1.3"
+    id("net.kyori.indra.git") version "3.1.3"
+    id("net.kyori.indra.publishing") version "3.1.3"
+    id("io.freefair.lombok") version "8.4"
```

Comment:
reference the version catalogue - i think the format is something like

`alias(libs.plugins.lombok)`